File: 6-set.py
ages = {10, 12, 15, 15, 17, 18, 18, 19, 20]}
print(ages)

ages.add(16)
ages.discard(20)

print(f"10:{10 in ages},\n16:{16 in ages},\n19:{19 in ages},\n20:{20 in ages}")

ages.clear()

print(ages)

File: lecture_notes/08 collections/6-set.js
'use strict';

const ages = new Set([10, 12, 15, 15, 17, 18, 18, 19, 20]);
console.log({ ages });

ages.add(16);
ages.delete(20);

console.log({
  10: ages.has(10),
  16: ages.has(16),
  19: ages.has(19),
  20: ages.has(20),
});

ages.clear();
console.log({ ages });
